integration/examples/nodejs/skaffold.yaml

```diff
@@ -4,6 +4,9 @@ build:
   artifacts:
   - image: gcr.io/k8s-skaffold/node-example
     context: backend
+    docker:
+      buildArgs:
+        SCRIPT: "{{.SCRIPT_ENV}}"
```

Comment:
i am guessing you meant "{{.SCRIPT}} here?

Reply:
Thanks for pointing it out.